### PR TITLE
bump up the prometheus and grafana versions 

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update the Prometheus version to v2.49.1
Update the Grafana version to v7.5.17

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: deploy-monitoring-services.md [hunk 1]
```diff
@@ -28,19 +28,19 @@ summary: 本文适用于手动部署 TiDB 监控报警系统的用户。假设 T
 {{< copyable "shell-regular" >}}
 
 ```bash
-wget https://download.pingcap.org/prometheus-2.27.1.linux-amd64.tar.gz
+wget https://download.pingcap.org/prometheus-2.49.1.linux-amd64.tar.gz
```
Comment (qiancai) [inline deploy-monitoring-services.md:31]: this link does not work

Comment (wuhuizuo) [reply deploy-monitoring-services.md:31]: Official Download Link： "https://github.com/prometheus/prometheus/releases/download/v${prometheus_ver}/prometheus-${prometheus_ver}.${prometheus_os}-${prometheus_arch}.tar.gz"

Comment (wuhuizuo) [inline deploy-monitoring-services.md:31]:
```suggestion
wget https://github.com/prometheus/prometheus/releases/download/v2.49.1/prometheus-2.49.1.linux-amd64.tar.gz
```

File: deploy-monitoring-services.md [hunk 2]
```diff
@@ -119,6 +119,16 @@ scrape_configs:
 ...
 ```
 
+如需告警，可额外下载告警文件 (可从 [pingcap/tidb](https://github.com/pingcap/tidb/blob/master/pkg/metrics/alertmanager/tidb.rules.yml)，[tikv/tikv](https://github.com/tikv/tikv/blob/master/metrics/alertmanager/tikv.rules.yml)，[tikv/pd](https://github.com/tikv/pd/blob/master/metrics/alertmanager/pd.rules.yml) 下载）。并在 prometheus 中添加如下配置：
```
Comment (qiancai) [inline deploy-monitoring-services.md:122]:
```suggestion
如需开启 [TiDB 集群报警规则](/alert-rules.md)，请单独下载 [TiDB](https://github.com/pingcap/tidb/blob/master/pkg/metrics/alertmanager/tidb.rules.yml)、[TiKV](https://github.com/tikv/tikv/blob/master/metrics/alertmanager/tikv.rules.yml)  和 [PD](https://github.com/tikv/pd/blob/master/metrics/alertmanager/pd.rules.yml) 的报警规则文件，并在 Prometheus 的配置文件中添加如下配置：
```

File: deploy-monitoring-services.md [hunk 3]
```diff
@@ -119,6 +119,16 @@ scrape_configs:
...
+
+```
```
Comment (qiancai) [inline deploy-monitoring-services.md:124]:
````suggestion
```ini
````

File: deploy-monitoring-services.md [hunk 4]
```diff
@@ -119,6 +119,16 @@ scrape_configs:
 ...
 ```
 
+如需开启 [TiDB 集群报警规则](/alert-rules.md)，请单独下载 [TiDB](https://github.com/pingcap/tidb/blob/master/pkg/metrics/alertmanager/tidb.rules.yml)、[TiKV](https://github.com/tikv/tikv/blob/master/metrics/alertmanager/tikv.rules.yml)  和 [PD](https://github.com/tikv/pd/blob/master/metrics/alertmanager/pd.rules.yml) 组件的报警规则文件，并在 Prometheus 的配置文件中添加如下配置：
```
Comment (qiancai) [inline deploy-monitoring-services.md:122]:
```suggestion
如需开启 [TiDB 集群报警规则](/alert-rules.md)，请单独下载 TiDB、TiKV 和 PD 组件的报警规则文件 [tidb.rules.yml](https://github.com/pingcap/tidb/blob/master/pkg/metrics/alertmanager/tidb.rules.yml)、[tikv.rules.yml](https://github.com/tikv/tikv/blob/master/metrics/alertmanager/tikv.rules.yml) 和 [pd.rules.yml](https://github.com/tikv/pd/blob/master/metrics/alertmanager/pd.rules.yml)，并在 Prometheus 的配置文件中添加如下配置：
```

File: deploy-monitoring-services.md [hunk 5]
```diff
@@ -119,6 +119,16 @@ scrape_configs:
...
 ```
 
+如需开启 [TiDB 集群报警规则](/alert-rules.md)，请单独下载 TiDB、TiKV 和 PD 组件的报警规则文件 [`tidb.rules.yml`](https://github.com/pingcap/tidb/blob/master/pkg/metrics/alertmanager/tidb.rules.yml)、[`tikv.rules.yml`](https://github.com/tikv/tikv/blob/master/metrics/alertmanager/tikv.rules.yml) 和 [`pd.rules.yml`](https://github.com/tikv/pd/blob/master/metrics/alertmanager/pd.rules.yml)，并在 Prometheus 的配置文件中添加如下配置：
+
+```ini
+rule_files:
+  - 'tidb.rules.yml'
+  - 'pd.rules.yml'
+  - 'tikv.rules.yml'
+  - 'tikv.accelerate.rules.yml'
```
Comment (xhebox) [inline deploy-monitoring-services.md:129]: 这个文件是来自`https://github.com/tikv/tikv/blob/master/metrics/alertmanager/tikv.accelerate.rules.yml`，我没写。可以提一下，让他想下什么文件就下哪个，比如还有tiflash之类的。

File: deploy-monitoring-services.md [hunk 6]
```diff
@@ -28,19 +28,19 @@ summary: 本文适用于手动部署 TiDB 监控报警系统的用户。假设 T
 {{< copyable "shell-regular" >}}
 
 ```bash
-wget https://download.pingcap.org/prometheus-2.27.1.linux-amd64.tar.gz
+wget https://github.com/prometheus/prometheus/releases/download/v2.49.1/prometheus-2.49.1.linux-amd64.tar.gz
 wget https://download.pingcap.org/node_exporter-v1.3.1-linux-amd64.tar.gz
-wget https://download.pingcap.org/grafana-7.5.11.linux-amd64.tar.gz
+wget https://download.pingcap.org/grafana-7.5.17.linux-amd64.tar.gz
```
Comment (wuhuizuo) [inline deploy-monitoring-services.md:33]: LGTM

File: deploy-monitoring-services.md [hunk 7]
```diff
@@ -28,19 +28,19 @@ summary: 本文适用于手动部署 TiDB 监控报警系统的用户。假设 T
...
 解压二进制包：
 
 {{< copyable "shell-regular" >}}
 
 ```bash
-tar -xzf prometheus-2.27.1.linux-amd64.tar.gz
+tar -xzf prometheus-2.49.1.linux-amd64.tar.gz
 tar -xzf node_exporter-v1.3.1-linux-amd64.tar.gz
-tar -xzf grafana-7.5.11.linux-amd64.tar.gz
+tar -xzf grafana-7.5.17.linux-amd64.tar.gz
```
Comment (wuhuizuo) [inline deploy-monitoring-services.md:43]: LGTM

File: deploy-monitoring-services.md [hunk 8]
```diff
@@ -67,7 +67,7 @@ cd node_exporter-v1.3.1-linux-amd64
 {{< copyable "shell-regular" >}}
 
 ```bash
-cd prometheus-2.27.1.linux-amd64 &&
+cd prometheus-2.49.1.linux-amd64 &&
```
Comment (wuhuizuo) [inline deploy-monitoring-services.md:70]: LGTM

**Review by wuhuizuo — APPROVED** [review]

**Review by kaaaaaaang — APPROVED** [review]

File: upgrade-tidb-using-tiup.md [hunk 9]
```diff
@@ -66,7 +66,7 @@ summary: TiUP 可用于 TiDB 升级。升级过程中需注意不支持 TiFlash
 - 支持 TiDB Binlog，TiCDC，TiFlash 等组件版本的升级。
 - 将 v6.3.0 之前的 TiFlash 升级至 v6.3.0 及之后的版本时，需要特别注意：在 Linux AMD64 架构的硬件平台部署 TiFlash 时，CPU 必须支持 AVX2 指令集。而在 Linux ARM64 架构的硬件平台部署 TiFlash 时，CPU 必须支持 ARMv8 架构。具体请参考 [6.3.0 版本 Release Notes](/releases/release-6.3.0.md#其他) 中的描述。
 - 具体不同版本的兼容性说明，请查看各个版本的 [Release Note](/releases/release-notes.md)。请根据各个版本的 Release Note 的兼容性更改调整集群的配置。
-- 升级 v5.3 之前版本的集群到 v5.3 及后续版本时，默认部署的 Prometheus 会从 v2.8.1 升级到 v2.27.1，v2.27.1 提供更多的功能并解决了安全风险。Prometheus v2.27.1 相对于 v2.8.1 存在 Alert 时间格式变化，详情见 [Prometheus commit](https://github.com/prometheus/prometheus/commit/7646cbca328278585be15fa615e22f2a50b47d06)。
+- 升级 v5.3 之前版本的集群到 v5.3 及后续版本时，TiUP 默认部署的 Prometheus 生成的 Alert 存在时间格式变化。该格式变化是从 Prometheus v2.27.1 开始引入的，详情见 [Prometheus commit](https://github.com/prometheus/prometheus/commit/7646cbca328278585be15fa615e22f2a50b47d06)。
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:69]:
```suggestion
- 升级 v5.3 之前版本的集群到 v5.3 及后续版本时，默认部署的 Prometheus 生成的 Alert 存在时间格式变化。该格式变化是从 Prometheus v2.27.1 开始引入的，详情见 [Prometheus commit](https://github.com/prometheus/prometheus/commit/7646cbca328278585be15fa615e22f2a50b47d06)。
```

**Review by xhebox — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Pull request merged**